.github/workflows/pr-platform.yml

```diff
@@ -12,12 +12,14 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - ubuntu-latest
+          - ubuntu-20.04
```

Comment:
this is going to change, but we want to explicitly support this version for the mmoc